### ISSUE TITLE
Use __author__ and __version__

### ISSUE THREAD

**Issue by Adrianzatreanu** [coala-analyzer/coala#2341]
Each would contain metadata about the bear

**Comment by AbdealiJK** [comment]
In https://github.com/coala-analyzer/coala/pull/2338/commits/5f1ad91c14ac6dc6bf7df58f125e1f39ed539964 @Adrianzatreanu added a bunch of variables in Bears like AUTHORS, MAINTAINERS, etc.

Normally, in python files, there are variables like `__author__`, `__version__`, etc. Which can directly be used to get this information. i.e. If a file "ABCBear.py" has a bear class "ABCBear", the class should auto use the `__author__` variables in the file for the AUTHOR class variable. And so on...

@Adrianzatreanu I think this should probably be a difficulty/medium ? Not sure if it's easy ...

**Comment by sils1297** [comment]
I agree, this might not be trivial and I'm pretty sure it's not important because it makes sense to have just all metadata plain and simple in a bear so it's a very optional thing. I'm pledging for a wontfix, accepting patches.

**Comment by Makman2** [comment]
maybe this is possible when doing an `import`^^ (maybe from `Bear` to avoid another module the user needs to import)
